feedback/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ issue_tracker: https://github.com/ueman/feedback/issues
 
 environment:
   sdk: '>=2.12.0 <3.0.0'
```

Comment:
We're now also able to bump the Dart version, but I guess that doesn't matter much for this library.